SRJPE/grunID
# This commit adds a way to modify the genetic id results.

The results is the most important table therefore as well as allowing users to modify this table I added a trigger and audit table to keep track of all changes that are made to it. This commit implements all this logic.

adds method for querying any result

(wip) adds docs to update function

(wip) adds function for viewing audits

## To Test

Pull latest run-id-db changes and re-spin up container to make migrations run. Use the new `update_genetic_run_id` function to update the run type for a sample. Use `view_genetic_id_audits` to verify audits are capture correctly.

closes #121
